proto/internalapi/central/compliance_operator.proto

```diff
@@ -32,23 +32,28 @@ message DisableComplianceRequest {
 // ApplyComplianceScanConfigRequest contains information about a compliance scan configuration to be applied to secured cluster.
 message ApplyComplianceScanConfigRequest {
     message BaseScanSettings {
-        string  db_record_id                = 1;
-        string  scan_name                   = 2;
+        string              db_record_id    = 1;
```

Comment:
Do we need this?  I was working through the scan schedule protos and had convinced my self that the name needed to be unique and that was sufficient.

Reply:
I will be adding ID label to the resource so that we know which resource maps to what in case a resource with the same name as previously existed resource is created.